frontend/src/components/SuggestionButtons/SuggestionButtonStyles.ts

```diff
@@ -0,0 +1,16 @@
+import { makeStyles, shorthands, tokens } from "@fluentui/react-components";
+
+export const SuggestionButtonStyles = makeStyles({
+    container: {
+        width: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        // center horizontally
+        alignItems: 'center',
+        // center vertically
```

Comment:
Nit: do we want to keep the commented code on lines 8 & 10?

Reply:
Removed.